libs/std/stdio.d

```diff
@@ -2,8 +2,15 @@
 module std.stdio;
 
 
-void writeln(string s)
-{
-    printf(s);
-    printf("\n");
+	
+void writeln(string s) {
+	//NOTE sdc cannot take the pointer form string constants
+	static immutable string formatString = "%.*s\n";
```

Comment:
This works because there's always a zero added to string literals at the end?  Does libd-llvm do this?